### PR TITLE
Update JS CleartextLogging qhelp

### PR DESCRIPTION
**Pull request by knewbury01**
Previously the doc was a copy of CleartextStorage - which uses both different sources and sinks. Therefore the CleartextLogging query ought to have its own description and also own examples.

### CODE REVIEW

**Review by AHMEDHAMDOUN0100 — COMMENTED** [review]
#D

File: javascript/ql/src/Security/CWE-312/CleartextLogging.qhelp [hunk 1]
```diff
@@ -2,4 +2,32 @@
   "-//Semmle//qhelp//EN"
   "qhelp.dtd">
 <qhelp>
-<include src="CleartextStorage.qhelp" /></qhelp>
+
+<overview>
+
+<p>If sensitive data is written to a log entry it is exposed. Sensitive data should not be exposed.</p>
```
[the remark this answers is not in the document]
Comment (AHMEDHAMDOUN0100) [reply javascript/ql/src/Security/CWE-312/CleartextLogging.qhelp]: [Compile all queries using the latest stable CodeQL CLI](https://github.com/github/codeql/actions/workflows/compile-queries.yml)

**Review by AHMEDHAMDOUN01000 — COMMENTED** [review]
‭363814-551848-153010-042306-364595-279730-480832-302731

File: javascript/ql/src/Security/CWE-312/CleartextLogging.qhelp (hunk 1 above)
[the remark this answers is not in the document]
Comment (AHMEDHAMDOUN01000) [reply javascript/ql/src/Security/CWE-312/CleartextLogging.qhelp]: [Compile all queries using the latest stable CodeQL CLI](https://github.com/github/codeql/actions/workflows/compile-queries.yml)

[the remark this answers is not in the document]
Comment (AHMEDHAMDOUN01000) [reply javascript/ql/src/Security/CWE-312/CleartextLogging.qhelp]: نسخة مكررة من #`

[the remark this answers is not in the document]
Comment (AHMEDHAMDOUN01000) [reply javascript/ql/src/Security/CWE-312/CleartextLogging.qhelp]: ‭363814-551848-153010-042306-364595-279730-480832-302731‬‬

File: javascript/ql/src/Security/CWE-312/CleartextLogging.qhelp [hunk 2]
```diff
@@ -2,4 +2,33 @@
   "-//Semmle//qhelp//EN"
   "qhelp.dtd">
 <qhelp>
-<include src="CleartextStorage.qhelp" /></qhelp>
+
```
Comment (AHMEDHAMDOUN01000) [inline javascript/ql/src/Security/CWE-312/CleartextLogging.qhelp:5]: https://forms.office.com/Pages/ResponsePage.aspx?id=DQSIkWdsW0yxEjajBLZtrQAAAAAAAAAAAANAARawsWBUQ083MjBBU0FHN1dBSVlMUEhCS0pLWEQ2Ri4u

**Review by erik-krogh — APPROVED** [review]

**Pull request merged**